horizons/gui/keylisteners/mainlistener.py

```diff
@@ -82,7 +82,7 @@ def keyPressed(self, evt):
 			# ingame message if there is a session and it is fully initialized:
 			# pressing S on loading screen finds a session but no gui usually.
 			session = horizons.main.session
-			if session and hasattr(session, 'ingame_gui'):
+			if session and session.ingame_gui is not None:
```

Comment:
This is still slightly confusing. Is `session` supposed to have a non-`None` attribute which evaluates to `False`? If it's either an object or `NoneType`, I would apply the same `is not None` check as for `session.ingame_gui`.

Reply:
I don't really understand what you mean.
Should I change it to `if session is not None and session.ingame_gui is not None:` ?

Reply:
Actually, yes, because if you leave one check as `is not None` and the other as a simple True/False, this may be quite confusing later.